spec/solver_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'solver'
```

Comment:
- [ ] Please replace `require_relative` with `require` because the rspec didn't find the `solver.rb` file if you import the solver file with require it will work properly  so kindly fix this issue because I got an error when I run you tests :slightly_smiling_face: 

```suggestion
require './solver'
```

Reply:
Thank you @aamirkhan2478